app/components/DataTable/components/DetailDialog/index.tsx

```diff
@@ -89,7 +89,9 @@ const ObjectAttributes = ({
               expandIcon={<ExpandMoreIcon />}
             >
               <Typography variant="subtitle2">
-                {`${idx}: ${mappedValIsArray ? '[' : '{'}`}
+                {!!mappedVal?.name ? `${mappedVal.name}` : ''}
+                {!!mappedVal?.username ? `${mappedVal?.firstName} ${mappedVal?.lastName}` : ''}
+                {`${!mappedVal?.username && !mappedVal?.name ? idx : ''}: ${mappedValIsArray ? '[' : '{'}`}
```

Comment:
this is supposed to be a generic component, adding all of these defeats the purpose

Reply:
Can I make them optional props?